server/database/config/fakeData.sql

```diff
@@ -0,0 +1,60 @@
+BEGIN;
+
+DROP TABLE IF EXISTS admin,cohort,student,project,studentProject CASCADE;
+
+CREATE TABLE admin 
+(
+    id SERIAL PRIMARY KEY NOT NULL ,
```

Comment:
You don't need to add ```NOT NULL``` to the ```PRIMARY KEY```
  ```suggestion
      id SERIAL PRIMARY KEY,
```